glass/galaxies.py

```diff
@@ -99,6 +103,13 @@ def redshifts_from_nz(
         samples from all populations.
 
     """
+    if warn:
+        warnings.warn(
+            "when sampling galaxies, redshifts_from_nz() is often not the function you"
+            " want. Try redshifts() instead. Use warn=False to suppress this warning.",
+            stacklevel=2,
+        )
```

Comment:
Maybe we don't need the `warn` keyword, and we should let the ignoring part be at the user end  (users can suppress warnings with something like `import warnings; warnings.filterwarnings("ignore")`)

Reply:
They can, with significantly more effort :)

Reply:
But, yeah, I agree that this is slightly muddying the separation of concerns. My reason for proposing the parameter is that I don't think many users would a priori know how to suppress a warning. So either we would have to let them figure it out by themselves, or explain it in the docs of the function, at which point the users would be blindly copying code. It seems easier to take the hit and add the flag ourselves.